src/highlighters/date.rs

```diff
@@ -6,17 +6,25 @@ use crate::theme::Style;
 use crate::types::Highlight;
 
 pub struct DateHighlighter {
-    date: String,
-    time: String,
-    zone: String,
+    date_ansi: Option<String>,
+    time_ansi: Option<String>,
+    zone_ansi: Option<String>,
```

Comment:
I opted for `Option<String>` as we can have a `hidden` style (field not displayed), and `disabled` style (field not styled)